Expecto.Tests/FocusedTests.fs

```diff
@@ -74,5 +74,31 @@ let all =
         result.ignored.Length ==? 19
 ]
 
+[<Tests>]
+let canDetectFocusedTest =
+  let localList = 
+    testList "local" [
+      ftestList "focused" [
+        testCase "test" (fun () -> ())
+      ]
+    ]
+
+  // check if we can fail on focused tests
+  if runTests { defaultConfig with failOnFocusedTests = true } localList <> 1 then
+    failwith "focused test check didn't fail"
+
+[<Tests>]
+let canRunIfNoFocusedTest =
+  let localList = 
+    testList "local" [
+      testList "focused" [
+        testCase "test" (fun () -> ())
+      ]
+    ]
+
+  // check if we pass if no focused tests exist
+  if runTests { defaultConfig with failOnFocusedTests = true } localList <> 0 then
+    failwith "focused test check didn't fail"
```

Comment:
Why not make it a normal test? This is code that executes on static module initialisation if I read the core correctly.